jep/1/README.adoc

```diff
@@ -0,0 +1,754 @@
+documenting the design decisions that have gone into Jenkins.
+Each JEP has an <<Author>>.
+The author of a JEP may or may not actually write the text of the JEP.
+Regardless, the JEP Author is responsible for building
+consensus for that JEP within the community and
+documenting dissenting opinions.
+
+Because the JEPs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+feature proposal footnoteref:[repo, https://github.com/jenkinsci/jep].
+
+
+=== JEP Types
+
+There are three kinds of JEP:
+
+. A **Standards Track** JEP describes a new feature or implementation
+ for Jenkins. It may also describe an interoperability or
+ backwards-compatibility standard which will be supported for the feature in
+ current versions of Jenkins, moving forward.
+. An **Informational** JEP describes a Jenkins design issue, or
+ provides general guidelines or information to the Jenkins community,
+ but does not propose a new feature. Informational JEPs do not
+ necessarily represent a Jenkins community consensus or
+ recommendation, so users and implementers are free to ignore
+ Informational JEPs or follow their advice.
+. A **Process** JEP describes a process surrounding Jenkins, or
+ proposes a change to (or an event in) a process. Process JEPs are like
+ Standards Track JEPs but apply to areas other than the Jenkins codebase
+ itself. They may propose an implementation, but not for what would be
+ generally considered the Jenkins codebase; they often require community
+ consensus; unlike Informational JEPs, they are more than recommendations, and
+ users are typically not free to ignore them. Examples include procedures,
+ guidelines, changes to the decision-making process, and changes to the tools
+ or environment used in Jenkins development. Any meta-JEP (JEP focusing on the
+ improvement of the Jenkins Enhancement Proposal process) is also considered a
+ Process JEP.
+
+=== JEP Terminology
+
+==== BDFL
+
+There are several references in this JEP to the "BDFL". This acronym stands for
+"Benevolent Dictator for Life", and refers to
+link:https://github.com/kohsuke[Kohsuke Kawaguchi],
+original creator of Jenkins.
+
+==== Editors
+
+The JEP editors are individuals responsible for managing the administrative
+and editorial aspects of the JEP workflow (e.g. assigning JEP numbers and
+changing their status). See <<editor-responsibilities, JEP Editor Responsibilities & Workflow>> for
+details. The current editors are:
+
+* link:https://github.com/rtyler[R. Tyler Croy]
+* link:https://github.com/oleg-nenashev[Oleg Nenashev]
+* link:https://github.com/bitwiseman[Liam Newman]
+* link:https://github.com/omehegan[Owen Mehegan]
+* link:https://github.com/orrc[Christopher Orr]
+
+JEP editorship is by invitation of the current editors. All of the JEP workflow
+can be conducted via the GitHub JEP Repository footnoteref:[repo] and pull
+requests.
+
+==== Author
```

Comment:
@oleg-nenashev  I'm going to delete your comment above as duplicate of this one, to centralize this discussion:

 > Rename to <<Champion>>? Author is a legal term, which contradicts with the next sentence (line 65)

I support unifying these terms into one since they are  currently used interchangeably in this document.   @rtyler  Do you have a preference as to which one we use?

Reply:
@bitwiseman no, go for what feels most comfortable to you :+1:

Reply:
"Sponsor" is quite ambiguous as well, but this term introduced by @bitwiseman solves my original concern about legal clashes. Hence it is RESOLVED